web/satellite/src/App.vue

```diff
@@ -36,6 +19,11 @@
   src: url("../static/fonts/montserrat_bold.ttf");
 }
 
+a{
+  cursor: pointer;
+  text-decoration: none;
```

Comment:
Do not remove decoration of links globally. This makes it less obvious what is a link. It's fine to remove these in specific places where there are other signifiers (like a button border).